packages/taquito-rpc/src/types.ts

```diff
@@ -11,6 +11,80 @@ export type DelegateResponse = string | null;
 
 export type OperationHash = string;
 
+interface Before {
+  value?: string;
+  node?: string;
+}
+
+interface After {
+  value?: string;
+  node?: string;
+}
+
+interface INodeExtender {
+  length: number;
+  segment: string;
+  proof: string;
+}
+
+interface OtherEltsInner {
+  value: any;
+  inode_extender?: INodeExtender;
+}
+
+interface OtherElts {
+  node?: [[string, { value: string } | { node: string }]];
```

Comment:
should it be [string, { value: string } | { node: string }][ ]
I don't think it is equivalent